### ISSUE TITLE
Warning running unit tests

### ISSUE THREAD

**Issue by hgezim** [rofrischmann/inline-style-prefixer#41]
I'm using [radium](https://github.com/FormidableLabs/radium) which relies on `inline-style-prefixer`.

Running my unit tests with [jest](https://github.com/facebook/jest), I get the following warning:

```
Either the global navigator was undefined or an invalid userAgent was provided. Using a valid userAgent? Please let us know and create an issue at https://github.com/rofrischmann/inline-style-prefixer/issues
```
Is there a way to disable this warning?

**Comment by ianobermiller** [comment]
https://github.com/FormidableLabs/radium/tree/master/docs/faq#how-can-i-get-rid-of-useragent-warnings-in-tests

**Comment by rofrischmann** [comment]
Seems like I'll probably add a note to that link in the repo to :) hope ians response helped :p

**Comment by hgezim** [comment]
Yup! Thanks both :)
